en/editor/project/index.md

```diff
@@ -68,6 +68,94 @@ The name of the compressed texture is only used for display. When adding a compr
 
 If you want to replace all the preset options currently used for image asset, you can move the mouse to the preset name, click the button to copy the ID to and search and replace in the project by yourself.
```

Comment:
If you want to replace all the preset options currently used for image asset, you can move the mouse to the preset name, click the button to copy the ID to and search and replace in the project by yourself.
-->
If all the preset options used for an image asset need to be replaced, move the mouse to the preset name, click the button to copy the ID to and search and replace it in the project manually.